commands_to_load/Here.py

```diff
@@ -0,0 +1,62 @@
+# Commands for finding who has access to certain channels.
+# Useful since the server size does not allow offline users to be listed
+#     in the sidebar
+
+from discord.ext import commands
+import discord
+import logging
+logger = logging.getLogger('wall_e')
+
+
+class Here():
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    def build_embed(members, channel):
+        # build response
+        embed = discord.Embed(type = "rich")
+        embed.title = "Users in **" + channel.name + "**"
+        embed.color = discord.Color.blurple()
+        embed.set_footer(text="brenfan", icon_url="https://i.imgur.com/vlpCuu2.jpg")
+        if len(members) == 0:
```

Comment:
please include some logging to indicate

1. How many members there are

1. What the title will be set to

Reply:
done